models/VolunteerModel.js

```diff
@@ -7,31 +7,59 @@ const VolunteerModel = new Schema(
     mode: { type: String, required: true },
     name: { type: String, required: true },
     mobile: { type: String, require: true },
+    act: { type: String, required: true },
+
+    availability: { type: String, required: false },
+    email: { type: String, required: false },
+    address: { type: String, required: false },
 
     region: { type: [String], required: true },
     pin: { type: String, required: true },
-    availability: { type: String, required: true },
 
-    email: { type: String, required: false },
-    dob: { type: Date, required: false },
-    gender: { type: String, required: false },
-
-    profession: { type: String, required: false },
-    qualification: { type: String, required: false },
-    address: { type: String, required: false },
+    services: {
+      type: [
+        {
+          id: { type: String, required: true },
+          values: {
+            type: [
+              {
+                id: { type: String, required: true },
+                attributes: { type: Object, required: false, default: {} },
+              },
+            ],
+            required: false,
+            default: [],
+          },
+        },
+      ],
+      required: false,
```

Comment:
I believe we can make services mandatory. Because a volunteer should minimum mention that on which service he could help.

Reply:
makes sense